### ISSUE TITLE
Choosing the default dashboard doesn't work

### ISSUE THREAD

**Issue by Kuzmin-Dmitry** [the-maux/tipboard#97]
The application ignores the set default dashboard

cat config/src/tipboard/app/properties.py :
```
...
LAYOUT_CONFIG = os.path.join(user_config_dir, 'projects.yaml')
...
```

http://localhost:8080/flipboard/getDashboardsPaths :
`{"paths": ["/project01", "/projects", "/project02", "/project03"], "names": ["Project 01", "Projects", "Project 02", "Project 03"]}`

http://localhost:8080/api/info :
`{"tipboard_version": "v0.1", "project_name": "tipboard-python3", "project_layout_config": "/home/app/src/tipboard/app/Config/projects.yaml", "redis_db": 4}
`

When you enter localhost:8080 "Project 01" dashboard (/project01) opens instead of Projects (/projects)

**Comment by the-maux** [comment]
Hello Dmitry,
**Once again**, you right, this features is not working since the migration to tipboard 2.0  
Don't hesitate to check the [new wiki about dashboard](https://github.com/the-maux/tipboard/wiki/Dashboard-Config-File), i know it's in W.I.P, but i'm trying to improve it  

I fixed this issue, try again in develop branch, the option with **LAYOUT_CONFIG**, is working again
It was fixed in #100 , try it and let me know :wink: 
I still need to improve the documentation about it

Thanks again for your support @Kuzmin-Dmitry, 
with your help, this project going to be ready really soon

**Comment by the-maux** [comment]
This is defined in the properties.json not in the in the properties.py but you can still override the value of course :smile: 
This is documented now, [here](https://github.com/the-maux/tipboard/wiki/Dashboard-Config-File)
